vmdb/app/views/layouts/listnav/_show_list.html.haml

```diff
@@ -1,88 +1,88 @@
+          %h4.panel-title
+            = link_to(_("My Filters"),
+                      "#collapseTwo",
+                      "data-toggle" => "collapse", 
+                      "data-parent" => "#accordion")
+          - if @panels["#{@view.db}_my_searches"].nil?
+            - @panels["#{@view.db}_my_searches"] = true
+
+        #collapseTwo.panel-collapse.collapse
+          .panel-body
+            %ul.nav.nav-pills.nav-stacked
+              - @my_searches.each do |search|
+                - if !@default_search.blank? && @default_search.name == search.name
+                  - if @edit && @edit[:expression] && ((@edit[:expression][:selected].nil? && @edit[:selected].nil?) || (!@edit[:expression][:selected].nil? && @edit[:expression][:selected][:id] == search.id.to_i && !@edit[:custom_search]))
+                    - li_class = "active"
+                  - else
+                    - li_class = ""
+                - elsif @edit && @edit[:expression] && ((@edit[:expression][:selected].nil? && search.id.to_i == 0) || (@edit[:expression][:selected] && @edit[:expression][:selected][:name] == search.name && !@edit[:custom_search]))
+                  - li_class = "active"
                 - else
-                  - bgcolor = ""
-              - elsif @edit && @edit[:expression] && ((@edit[:expression][:selected].nil? && search.id.to_i == 0) || (@edit[:expression][:selected] && @edit[:expression][:selected][:name] == search.name && !@edit[:custom_search]))
-                - bgcolor = "#d5d5d5"
-              - else
-                - bgcolor = ""
-              = link_to(image_tag("/images/icons/new/link_internal.gif", :width => 16, :height => 16, :align => 'left', :border => '0', :style => "background-color:#{bgcolor}"),
-                {:action => 'listnav_search_selected', :id => search.id, :button => "apply"},
-                :remote  => true,
-                :title   => _("Apply this filter"),
-                :onclick => "return miqCheckForChanges()")
-              - if !@default_search.blank? && @default_search.name == search.name
-                - search_name = "#{search.description} #{image_tag("/images/layout/set-default-sm.png", :valign => 'middle', :width => 16, :height => 16, :border => '0', :style => 'background-color: %s;') % bgcolor}"
-              - else
-                - search_name = search.description
-              = link_to(search_name.html_safe,
-                {:action => 'listnav_search_selected', :id => search.id, :button => "apply"},
-                :remote  => true,
-                :title   => _("Apply this filter"),
-                :onclick => "return miqCheckForChanges()",
-                :style   => "background-color:#{bgcolor}")
+                  - li_class = ""
+                - if !@default_search.blank? && @default_search.name == search.name
+                  - search_name = search.description
+                - else
+                  - search_name = search.description
```

Comment:
why do we have the `if` here when the positive branch is the same as the negative one?